_projects/2024/100452420/100452420.Rmd

```diff
@@ -0,0 +1,364 @@
+  annotate("segment", x = as.Date("2007-01-25"), xend = as.Date("2007-01-25"), y = 15, yend = 20,
+           linetype = "dotted", color = "#3093CF", size = 0.7) +
+  annotate("segment", x = as.Date("2011-05-20"), xend = as.Date("2011-05-20"), y = 17, yend = 20,
+           linetype = "dotted", color = "#3093CF", size = 0.7) +
+  annotate("segment", x = as.Date("2011-10-01"), xend = as.Date("2011-10-01"), y = 14, yend = 20,
+           linetype = "dotted", color = "#3093CF", size = 0.7) +
+  annotate("segment", x = as.Date("2017-09-01"), xend = as.Date("2017-09-01"), y = 15, yend = 20,
+           linetype = "dotted", color = "#3093CF", size = 0.7) +
+  annotate("segment", x = as.Date("2018-09-01"), xend = as.Date("2018-09-01"), y = 16, yend = 20,
+           linetype = "dotted", color = "#3093CF", size = 0.7) +
+  annotate("curve", x = as.Date("2002-01-01"), y = 1, xend = as.Date("2002-06-01"), yend = 1,
+           curvature = 0,
+           arrow = arrow(length = unit(0.1, "cm"), type = "closed"),
+           color = "black",
+           size = 0.4) +
+
+  annotate("curve", x = as.Date("2005-05-01"), y = 18, xend = as.Date("2006-01-01"), yend = 20,
+           curvature = -0.3,
+           arrow = arrow(length = unit(0.1, "cm"), type = "closed"),
+           color = "black",
+           size = 0.4) +
+
+  annotate("curve", x = as.Date("2017-07-01"), y = 18, xend = as.Date("2018-04-01"), yend = 20,
+           curvature = -0.3,
+           arrow = arrow(length = unit(0.1, "cm"), type = "closed"),
+           color = "black",
+           size = 0.4) +
+
+  annotate("curve", x = as.Date("2013-07-01"), y = 2, xend = as.Date("2013-04-01"), yend = 1.2,
+           curvature = -0.3,
+           arrow = arrow(length = unit(0.1, "cm"), type = "closed"),
+           color = "black",
+           size = 0.4) + 
+  theme(plot.margin = margin(5, 0, 0, 0))
+
+```
+
+
+The data for the tournaments was collected from [this website](https://en.wikipedia.org/wiki/Serena_Williams_career_statistics), but it was incomplete. I had to create an Excel file and complete the table with all the years and tournaments she played in.
+
+The libraries used for the improvements were patchwork (to join the two tables) and readxl (to read the Excel file).
+
+After cleaning the data and selecting relevant columns, I created the plot, assigning the color orange to tournaments she won, grey to those she did not play, and blue to those she played but did not win. I also adjusted the margins using theme(plot.margin) and ordered the tournaments in chronological order.
+
+Lastly, I joined both graphs, adjusting the heights so that the years matched in both tables.
+
+
+```{r, layout = "1-body-outset", fig.width= 12, fig.height= 7}
+
+library(patchwork)
+library(readxl)
+serena_slams <- read_xlsx("C:/Users/diegu/Desktop/Master in Computational Social Sciences/data visualization/replicate plot/dataviz/_projects/2024/100452420/SerenaWilliams_slams_data.xlsx")
```

Comment:
Ah, you use the excel file here. Then, remove the path. The name of the file is enough, because it's right here. The same with the CSVs.